### PR TITLE
Update with new Hyperparameter tuning

### PR DESCRIPTION
**Pull request by Rishi2003Das**
Change in Hyperparameter

### CODE REVIEW

**Comment by birm** [comment]
You increased the learning rate and number of epochs? Do you have loss/acc numbers to show for this? This would also involve the weights being different, right?

**Comment by Rishi2003Das** [comment]
Yes  As you said I will add the code for checking the accuracy and loss to show the best possible outcome from finetuning the hyperparameters. Yeah, this will surely affect the weights.

**Comment by Rishi2003Das** [comment]
Hi @birm , I am facing a problem making the necessary changes as I don't have access to the dataset. Can you please direct me on how can I use the dataset and run the code in my local system to make the necessary changes?
